pandas/io/formats/format.py

```diff
@@ -90,7 +91,7 @@
             The subset of columns to write. Writes all columns by default.
         col_space : %(col_space_type)s, optional
             %(col_space)s.
-        header : bool, optional
+        header : bool or sequence, optional
```

Comment:
Updated the doc too, I am not sure how docs are getting generated, hope this is enough for updating docs.
@WillAyd @simonjayhawkins We are all done (if doc changes is enough). You can review and merge

Reply:
the `to_html` method shares this docstring and does not support column label renaming using the header parameter.

apply this change to `to_string` _only_

Reply:
How do I do that? Sorry, never did docstrings.

Reply:
OK, figured it out after a bit of head scratching.